dbaccess/block.go

```diff
@@ -32,37 +32,20 @@ func StoreBlock(context Context, block *util.Block) error {
 		return errors.Errorf("block %s already exists", hash)
 	}
 
-	// Save a reference to the current block location in case
-	// we fail and need to rollback.
-	previousBlockLocation := db.CurrentStoreLocation(blockStoreName)
-	rollback := func() error {
-		return db.RollbackStore(blockStoreName, previousBlockLocation)
-	}
-
-	// Write the block's bytes to the block store and rollback
-	// if there's an error.
+	// Write the block's bytes to the block store
 	bytes, err := block.Bytes()
 	if err != nil {
 		return err
 	}
 	blockLocation, err := db.AppendToStore(blockStoreName, bytes)
 	if err != nil {
-		rollbackErr := rollback()
```

Comment:
You still want to rollback, no?

Reply:
Rollbacks are now generic over any data store, and such are implemented inside ffldb.